x-pack/plugins/cases/server/common/models/case_with_comments.ts

```diff
@@ -331,6 +338,38 @@ export class CaseCommentModel {
     await this.params.services.alertsService.updateAlertsStatus(alertsToUpdate);
   }
 
+  private async updateAlertsSchemaWithCaseInfo(alertAttachments: CommentRequestAlertType[]) {
+    try {
+      const alerts = getAlertInfoFromComments(alertAttachments);
```

Comment:
To be extra safe we might want to check for an empty alertId and index similar to what we do here: https://github.com/elastic/kibana/blob/main/x-pack/plugins/cases/server/services/alerts/index.ts#L104

I can't remember if we allow empty values in the create comment request though.